### PR TITLE
Separate the renders of the map

### PR DESCRIPTION
**Pull request by flomillot**
TS 1691

Signed-off-by: Florent MILLOT <millotflo@gmail.com>

### CODE REVIEW

File: src/components/network/geo-data.js [hunk 1]
```diff
@@ -115,6 +115,7 @@ export default class GeoData {
             ];
         }
 
+        // why it's not at the start of the function ?
```
Comment (etiennehomer) [inline src/components/network/geo-data.js:118]: Yes it can be refacto with the test at the start

File: src/components/study-pane.js [hunk 2]
```diff
@@ -262,6 +262,7 @@ const StudyPane = ({
                                 {/* TODO do not display if study does not exists or do not fetch geoData if study does not exists */}
                                 <NetworkMapTab
                                     /* TODO do we move redux param to container */
+                                    /* why passing params as props if it's stored in redux and only used by this component ? */
```
Comment (etiennehomer) [inline src/components/study-pane.js:265]: I think there's no reason

File: src/components/network/geo-data.js (hunk 1 above)
Comment (flomillot) [reply src/components/network/geo-data.js:118]: Ok I'll do it in an other PR

File: src/components/network/network-map.js [hunk 3]
```diff
@@ -445,9 +451,10 @@ const NetworkMap = (props) => {
 };
 
 NetworkMap.defaultProps = {
-    mapEquipments: null,
+    network: null,
+    substations: [],
+    lines: [],
```
Comment (etiennehomer) [inline src/components/network/network-map.js:456]: You can remove substations and lines nope ?

**Review by jonenst — APPROVED** [review]

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**